README.md

```diff
@@ -65,6 +65,31 @@ Currently the waiting period is as follows:
 
 ## FAQ
 
+#### Do I need to register a package to install it?
+
+No, you can simply do `using Pkg; Pkg.add(url="https://github.com/JuliaLang/Example.jl")`
+to e.g. install the package `Example.jl`, even if it was not registered.
+
+#### Should I register my package?
+
+If you think your package might be useful to others, or provide functionality other
+registered packages might want to rely on, go for it!
```

Comment:
Saying registered pacakge here is a bit odd.
I think i kniw what yoiu are getting at, but i think clearer not to say that.

Reply:
should it be just "other packages"?

Reply:
Also some how explain that it needs to be registered to be a indirect dependency

Reply:
I think I was getting at the idea that registering is opting-in to the package ecosystem dependency tree, so if you think it's a useful leaf ("might be useful to others") or node (other packages might want to depend on it) then you should register. (And to favor those reasons over other reasons like "I want to claim the namespace" or "only I use it but I don't like copying the Manifest around)